src/actions/pause.js

```diff
@@ -10,6 +10,7 @@ import type { Pause, Frame, Expression, Grip } from "../types";
 import type { ThunkArgs } from "./types";
 
 type CommandType = { type: string };
+type frameIdType = string | null;
```

Comment:
Hmm, i don't think so actually. `null` has special meaning for the frameId i believe. So it's not like checking for the presence of frameTypeId.